### ISSUE TITLE
integrations to the full-calendar plugin

### ISSUE THREAD

**Issue by pcause** [sandorex/set-in-obsidian-plugin#3]
Since you use full calendar, perhaps this could work as a calendar source for that plugin.  that way I can have external calendars and this for my local events in the same calendar.

**Comment by sandorex** [comment]
If they agree im all for it

Currently im low on free time so cant promise anything

**Comment by sandorex** [comment]
You can actually already add [google calendar](https://fullcalendar.io/docs/google-calendar) and [icalendar feed](https://fullcalendar.io/docs/icalendar) into the plugin by editing the calender config in dev options, i completely forgot i exposed that in the settings 😅

**Comment by pcause** [comment]
for me ics support would be what I need, but others want google calendar.  that is why i suggest seeing if the fill-caneldar person could work with you as he has that support and you add terrific additional capability.

**Comment by sandorex** [comment]
I am going to add ICS support and google calendar soon, it's already builtin in into FullCalendar library i just need to add some extra packages but the setup will be a bit manual with the token and config

I do not think we will be able to work together, i've made this plugin cause his did not parse events from list items which is how i want to use the plugin

**Comment by mayurankv** [comment]
Hi, I understand that you do not want to work with his plugin. Having said this, would it not be possible to expose the calendar of information you are rendering to his plugin as a 'calendar'. I suggest this because any user of his plugin and your plugin would now have two seperate conflicting calendar views whilst if your plugin was able to expose the source of events that are rendered in the calendar view, it could instead be rendered in his plugin as well resulting in only one block. Since I don't believe you are planning to supercede/replace all the functionality in that plugin, I think this would be really beneficial for any users of both. Please do let me know if I've been unclear about anything!

**Comment by mayurankv** [comment]
Ok thanks for the clarification! Sorry for misunderstanding your reasons for the separate implementation.
Would this be something you were willing to consider at a later stage if they did open up an API?

**Comment by sandorex** [comment]
> Would this be something you were willing to consider at a later stage if they did open up an API?

Definitely, but would probably be a separate smaller plugin focused solely on parsing nothing else

**Comment by mayurankv** [comment]
Fingers crossed that comes through then. I personally would love a plugin that makes it easy to add quick listed tasks to the calendar setup!
